### DISCUSSION TITLE
migrating-from-utterances-to-giscus-comments/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#8] [Comments]
# migrating-from-utterances-to-giscus-comments/

Since migrating from Wordpress to Jekyll, this blog (and my non-tech blog) has used Utterances for comments, which creatively uses GitHub issues for comment hosting. However, since then GitHub has released the “Discussions” feature which… is a much better fit! Luckily, Giscus is a way to use these, and can easily be migrated to.

https://blog.jakelee.co.uk/migrating-from-utterances-to-giscus-comments/

**Comment by JakeSteam** [comment]
Here's an example comment via Giscus!

**Reply by JakeSteam** [reply]
Here's an example reply via Giscus!

**Comment by anittasaju1996** [comment]
Great Blog! Thanks

**Comment by thomaslevesque** [comment]
Thanks for this. I've been thinking about migrating for a while. Seems pretty simple!
